packages/babel-parser/test/fixtures/typescript/class/static-asi/input.ts

```diff
@@ -0,0 +1,22 @@
+class A {
+    static
```

Comment:
Can you add a test for `static` followed by a modifier and then ASI?
```js
class A {
  static readonly
  foo
}
```
should be a static field called `readonly` and an instance field called `foo`